src/multiple-choice/components/runtime.tsx

```diff
@@ -39,6 +43,19 @@ export const Runtime: React.FC<IProps> = ({ authoredState, interactiveState, set
     // Question can be scored if it has at least one correct answer defined.
   const isScorable = !!authoredState.choices && authoredState.choices.filter(c => c.correct).length > 0;
 
+  useCustomMessages((msg: ICustomMessage) => {
+    if (msg.type === "teacher-edition:showCorrectOverlay") {
+      setShowCorrect(true);
+      setShowDistractor(false);
+    } else if (msg.type === "teacher-edition:showDistractorOverlay") {
+      setShowDistractor(true);
+      setShowCorrect(false);
+    } else if (msg.type === "teacher-edition:hideOverlay") {
+      setShowCorrect(false);
+      setShowDistractor(false);
+    }
```

Comment:
With three branches I'd probably opt for a switch statement, but it's a matter of preference.

Reply:
I switched to a switch (that sounds odd when said out loud).